librz/core/cmd/cmd_api.c

```diff
@@ -31,6 +31,11 @@
 #define MIN_SUMMARY_WIDTH    6
 #define MAX_RIGHT_ALIGHNMENT 20
 
+// Based on Config and Line order choses the vertical line shape
+static inline const char *get_vertical_line(int idx, int len, bool scr_curvy, bool scr_utf8) {
+	return (((idx == 0) && (scr_curvy == true)) ? RUNE_CURVE_CORNER_TL : (((idx == len - 1) && (scr_curvy == true)) ? RUNE_CURVE_CORNER_BL : ((scr_utf8 == true) ? RUNE_LINE_VERT : "| ")));
```

Comment:
this is too complicated.

1. don't use `signed int` for enums. use `ut32` or `size_t` which are unsigned.
2. Split this into if-else statement.
3. Don't test booleans, just use them as they are, example `if (scr_curvy) {` ....